authors/apps/authentication/tests/test_register.py

```diff
@@ -49,3 +53,37 @@ def test_api_cannot_create_a_user_with_password_lessthan_eight_characters(self):
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
         self.assertIn("Ensure this field has at least 8 characters.",
                       response.data["errors"]["password"])
+
+    def test_api_cannot_create_user_without_alphanumeric_password(self):
+        response = self.client.post(
+            register_url,
+            { "user": user_with_a_non_numeric_password},
+            format="json")
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+        self.assertIn("password must contain a numeric character", response.data["errors"]["password"])
```

Comment:
Please make sure that lines do not exceed 79 characters. Same for lines 71, 80 and 89.

Reply:
I have fixed this

Reply:
May I know how you've fixed it because as seen in the picture the line is 97 characters long.
<img width="852" alt="Screenshot 2019-04-10 at 14 33 32" src="https://user-images.githubusercontent.com/40039818/55875555-ffbbc680-5b9d-11e9-92b2-e45b9e013697.png">

Reply:
sorry, I hadnt then but I have now